systemtests/tests/py2plug-dir/python-modules/bareos-dir-test.py

```diff
@@ -19,59 +19,18 @@
 #
 # Author: Tobias Plum
 #
-from bareosdir import *
+import bareosdir
+import BareosDirWrapper
+from BareosDirWrapper import *  # noqa
```

Comment:
Can this * import be avoided?

Reply:
Not really, we could use
`from BareosDirWrapper import parse_plugin_definition, handle_plugin_event`
but that's not much nicer, and flake8 would still complain.

For a better solution which would only need the `import BareosDirWrapper` (or may be no wrapper module at all) we would have to change bareosdir.cc (may be also pythton-dir.cc), as they lookup  the functions in the module namespace. But that's another story, I don't think that we should do that within this PR.